e2e/script/e2e_init.js

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/* eslint-disable */
+
+require('shelljs/global');
+
+const argv = require('yargs').argv;
+const spawn = require('child_process').spawn;
+
+const retryExec = require('./retry_exec');
+const timesToRetry = argv.retries || 1;
+
+let SERVER_PID;
+let APPIUM_PID;
+let exitCode;
+
+// packages
+const APPIUM = 'appium@1.12.1';
+const WD = 'wd@1.11.2';
+const TAPE = 'tape@4.10.1';
+const TAP_SPEC = 'tap-spec@5.0.0';
+
+try {
+    if (argv.install) {
+        installTestFramework();
+    }
+
+    startPackagerServer();
+    startAppiumServer();
+
+    if (argv.android) {
+        testAndroid();
+    }
+
+    exitCode = 0;
+
+    describe('Success: completed E2E tests!');
+} catch {
+    describe('Failed: error encountered');
+} finally {
+    stopPackagerServer();
+    stopAppiumServer();
+}
+
+exit(exitCode);
+
+// Helper functions
+
+function describe(message) {
```

Comment:
I'd rename this. I was confused when I saw it used above because I thought it was coming from mocha/jest.

Reply:
Changed to `logStep`.